systems/WikibaseRepo/09-Architecture_Decisions.md

```diff
@@ -7,23 +7,3 @@ Since 2018 architecture decisions concerning Wikibase are recorded as ADRs. They
 ## Early architecture decisions
 
 This section contains retrospectively documented decisions that were made before the practice of recording them in ADRs was established. Some knowledge of the decision context likely went missing over time, so the reasoning is speculative to some degree.
-
-### Developing Wikibase Repo as a MediaWiki extension
```

Comment:
Going back to arc42's docs, it definitely makes more sense to move this whole explanation to the Solution Strategy section. :+1: